### PR TITLE
[ACR] Update table columns and ordering

### PR DESCRIPTION
**Pull request by djyou**
- Mostly restructure `_format.py` for better isolation and readability.
 - Fix ordering of columns (to be consistent with Azure Portal).
- Fix datetime formatting (remove milliseconds, see following as an example).
```
ID                                    ACTION    IMAGE          HTTP STATUS    TIMESTAMP
------------------------------------  --------  -------------  -------------  --------------------
42f07d41-d640-4c94-8030-46abb11462ea  ping                     200 OK         2018-04-06T19:02:04Z
dcd1be7d-97bc-4bbe-b12b-072c666f1946  delete    python         200 OK         2018-03-27T00:48:22Z
```
 - Add/Remove columns for a few commands (see following)

`acr list` (remove `STATUS`)
```
NAME               RESOURCE GROUP    LOCATION       SKU       LOGIN SERVER                  CREATION DATE         ADMIN ENABLED
-----------------  ----------------  -------------  --------  ----------------------------  --------------------  ---------------
```
`acr webhook list` (remove `RESOURCE GROUP`)
```
NAME        LOCATION       ACTIONS             SCOPE       STATUS
----------  -------------  ------------------  ----------  --------
```
`acr replication list` (remove `RESOURCE GROUP`, add `PROVISIONING STATE` and `STATUS`)
```
NAME           LOCATION       PROVISIONING STATE    STATUS
-------------  -------------  --------------------  --------
```
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6040
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-acr/HISTORY.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.23
 ++++++
+* Update table columns and ordering.
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acr/HISTORY.rst]: Change version to 2.0.24 and update setup.py.
The build for 2.0.23 has already been created.

**Comment by djyou** [comment]
@derekbekoe Comments addressed, could you please review/merge? Thanks.

**Pull request merged**